app/models/vm_or_template/operations/relocation.rb

```diff
@@ -60,6 +60,18 @@ def migrate(host, pool = nil, priority = "defaultPriority", state = nil)
     raw_migrate(host, pool, priority, state)
   end
 
+  def migration_status
+    miq_custom_get(:migration_status)
+  end
+
+  def reset_migration_status
```

Comment:
we don't need `reset` method

Reply:
We do or the migration process would be not correct when we run in next time for the same vm.